### PR TITLE
map route + fixes

### PR DESCRIPTION
**Pull request by JaskRendix**
PR addresses the missing routes and duplicates.

### CODE REVIEW

File: mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po [hunk 1]
```diff
@@ -804,6 +816,21 @@ msgid "welcome_location_town"
...
+msgstr "From ${{map_name}} to ${{south}}"
+
+msgid "here_to_west"
+msgstr "From ${{map_name}} to ${{west}}"
+
+msgid "here_to_east"
+msgstr "From ${{map_name}} to ${{east}}"
+
+msgid "alert_route"
+msgstr "Rehabilitation will begin as soon as possible. \n Please stop nagging us."
```
Comment (kerizane) [inline mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po]: You are about this one?

Comment (JaskRendix) [reply mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po]: Originally:
`msgstr "Route 3: Rehabilitation will begin as soon as possible. \n Please stop nagging us."`
Now the sign shows "From XXX to Route 3", then it appears another dialogue with "alert_route"
Yeah, probably wrong position, I'm going to move it close the route3 dialogues.

**Comment by JaskRendix** [comment]
Luckily I opened the issue, it was useful to confirm my doubts.

File: mods/tuxemon/maps/player_house_downstairs.tmx [hunk 2]
```diff
@@ -147,15 +147,15 @@
...
+    <property name="act10" value="clear_variable scene"/>
+    <property name="act11" value="clear_variable aeble_talk"/>
     <property name="act2" value="npc_face npc_mom,right"/>
     <property name="act3" value="translated_dialog ohnoes"/>
     <property name="act4" value="wait 0.7"/>
     <property name="act5" value="translated_dialog ohnoes2"/>
     <property name="act6" value="npc_face npc_mom,left"/>
     <property name="act8" value="unlock_controls"/>
     <property name="act9" value="set_variable can_exit_town:yes"/>
-    <property name="act10" value="clear_variable scene"/>
```
Comment (kerizane) [inline mods/tuxemon/maps/player_house_downstairs.tmx]: Leave these in numerical order, please.

Comment (JaskRendix) [reply mods/tuxemon/maps/player_house_downstairs.tmx]: As you wish, it was Tiled, I'll drop this file.

File: mods/tuxemon/maps/leather_town.tmx [hunk 3]
```diff
@@ -206,8 +206,8 @@
   </object>
   <object id="90" name="Sign: Route 3" type="event" x="544" y="16" width="16" height="16">
    <properties>
-    <property name="act10" value="translated_dialog spyder_route3_sign"/>
-    <property name="act20" value="translated_dialog leather_town_sign_leather_route"/>
+    <property name="act10" value="translated_dialog alert_route"/>
```
Comment (kerizane) [inline mods/tuxemon/maps/leather_town.tmx]: Why did you change the name to alert_route?

Comment (JaskRendix) [reply mods/tuxemon/maps/leather_town.tmx]: I had another idea. I'm going to change it back

**Review by kerizane — APPROVED** [review]
